.github/workflows/code_analysis.yaml

```diff
@@ -23,18 +23,10 @@ jobs:
                         name: Coding Standard README Sync
                         run: 'php ci/check_coding_standard_readme_sync.php'
```

Comment:
@todo replace with https://github.com/symplify/symplify/issues/2233 and move to ci_review.yaml workflow